internal/plugin.go

```diff
@@ -1237,6 +1240,26 @@ func assertValidPlacement(
 		}
 	}
 
+	if len(placement.ClusterSelectors) > 0 && len(placement.ClusterSelector) > 0 {
+		return fmt.Errorf("cannot use both clusterSelector and clusterSelectors in placement config " +
+			"(clusterSelector is recommended since it matches the actual placement field)")
+	}
+
+	// Determine which selectors to use
+	var resolvedSelectors map[string]interface{}
+	if len(placement.ClusterSelectors) > 0 {
+		resolvedSelectors = placement.ClusterSelectors
+	} else if len(placement.ClusterSelector) > 0 {
+		resolvedSelectors = placement.ClusterSelector
+	} else if len(placement.LabelSelector) > 0 {
+		resolvedSelectors = placement.LabelSelector
+	}
+
+	_, err := p.generateSelector(resolvedSelectors)
+	if err != nil {
+		return fmt.Errorf("invalid selectors in placement: %w", err)
```

Comment:
Could you use the `path` method argument in the error message so that the user knows where the invalid selector is?